sherpa/utils/__init__.py

```diff
@@ -119,6 +120,7 @@
 
 
 class NoNewAttributesAfterInit(object):
+
```

Comment:
https://github.com/hhatto/autopep8/issues/194

> As referenced in issues #83, #88, and #178, an earlier version of PEP 257 made the weird recommendation of inserting a blank line before the docstring of a class.
> 
> However, this recommendation has since been removed. The current PEP 257 only recommends adding a blank line after the docstring, not before:
> 
> Insert a blank line after all docstrings (one-line or multi-line) that document a class -- generally speaking, the class's methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line.
> (The commit that changed it: 9b715d8246db)
> 
> As far as i understand, autopep8's E309 fixer does not currently distinguish between docstrings and other class members when inserting blank lines, so this issue cannot be worked around with --ignore=E309 alone, if you still want autopep8 to insert blank lines before non-docstring members.